lib/modules/manager/bundler/artifacts.spec.ts

```diff
@@ -681,19 +470,18 @@ describe('modules/manager/bundler/artifacts', () => {
               'docker run --rm --name=renovate_sidecar --label=renovate_child ' +
               '-v "/tmp/github/some/repo":"/tmp/github/some/repo" ' +
               '-v "/tmp/cache":"/tmp/cache" ' +
+              '-e BUNDLE_GEMS___PRIVATE__COM ' +
               '-e GEM_HOME ' +
               '-e CONTAINERBASE_CACHE_DIR ' +
               '-w "/tmp/github/some/repo" ' +
               'ghcr.io/containerbase/sidecar' +
               ' bash -l -c "' +
               'install-tool ruby 1.2.0' +
               ' && ' +
-              'install-tool bundler 1.3.0' +
+              'install-tool bundler 2.3.5' +
```

Comment:
What caused this line to change in the test?

Reply:
This is GitHub diff pranking us. I actually didn't remove these lines:
https://github.com/renovatebot/renovate/pull/33339/files#diff-5421e221c5a44026d537b2f1168b1ce07db041069cba49bf2ba6462848e6c5cfL438-L489

So, because they're too similar to the lines I removed, it is showing a bogus visualization.

Happy new year! :tada: